app/controllers/application_controller.rb

```diff
@@ -2,4 +2,12 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery with: :exception
 
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
```

Comment:
A long-standing, really pointless argument, but, do have a look at `protected` vs `private` during your commute one day when you have the spare time. Its really quite amusing.